gitpod-io/gitpod
# docker-up: Enable rootless mode only in cgroup v1

## Description
Enable rootless mode only in cgroup v1 because docker requires systemd when using rootless mode in cgroup v2.


## Related Issue(s)

Fixes #9776

## How to test

Check the following in cgroup v1/v2

1. Open a workspace
2. Run the below command on the workspace
```console
$ docker run --rm --cpus 1 --memory 32m --name stress alexeiled/stress-ng --cpu=2
```
3. Open another console, and run the below command and check if the limit is working
```console
$ docker stats
```

## Release Notes

```release-note
Check the following in cgroup v1/v2
Eliminate dockerd rootless mode in cgroup v2
```

## Documentation


## Werft options:
